app_dart/test/service/luci_build_service_test.dart

```diff
@@ -584,15 +584,18 @@ void main() {
       );
       slug = RepositorySlug('flutter', 'cocoon');
     });
+
     test('Cancel builds when build list is empty', () async {
       when(mockBuildBucketClient.batch(any)).thenAnswer((_) async {
         return const BatchResponse(
           responses: <Response>[],
         );
       });
       await service.cancelBuilds(pullRequest, 'new builds');
+      // why is this getting called twice in this test?
       verify(mockBuildBucketClient.batch(any)).called(1);
```

Comment:
I am still working on figuring out why this is getting called twice and not once. For some reason the batch function is getting called two times and not once.